Jenkinsfile

```diff
@@ -34,11 +35,11 @@ node('imf2public') {
     }
 }
 
-def notifyFailed() {
-    emailext (
-        subject: "FAILED: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]'",
-        body: """<html><body><p>FAILED: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]':</p>
-            <p>Check console output at "<a href="${env.BUILD_URL}">${env.JOB_NAME} [${env.BUILD_NUMBER}]</a>"</p></html></body>""",
-        to: 'datamaps@gov.bc.ca'
-    )
-}
+// def notifyFailed() {
```

Comment:
Was this commented-out code left by mistake? If not I'd add a comment above saying why it's commented out

Reply:
Again, my bad not squashing. I commented out to avoid notifications during testing. it's uncommented.